bin/webpack.js

```diff
@@ -168,7 +170,23 @@ yargs.parse(process.argv.slice(2), (err, argv, output) => {
 		argv["display"] = "verbose";
 	}
 
-	var options = require("./convert-argv")(yargs, argv);
+	try {
+		Error.stackTraceLimit = 30;
+		var options = require("./convert-argv")(yargs, argv);
+	} catch(err) {
+		var stack = errorHelpers.cutOffAfterFlag(err.stack, "webpack.config");
```

Comment:
The user could provide any filename for the webpack.config.js via `--config asdf.js`

Reply:
I'll go with the marker function approach then, for now, it will include some traces from `Module` module, but will work with any file name